content/telegraf/v1/install.md

```diff
@@ -100,182 +99,167 @@ To manually install the Debian package from a `.deb` file:
-```
+The `telegraf` configuration file is installed at `/etc/telegraf/telegraf.conf`.
 
-For more advanced configuration details, see the
-[configuration documentation](/telegraf/v1/administration/configuration/).
 {{% /tab-content %}}
 <!---------- BEGIN SLES & openSUSE ---------->
 {{% tab-content %}}
-There are RPM packages provided by openSUSE Build Service for SUSE Linux users:
+The openSUSE Build Service provides RPM packages for SUSE Linux.
 
-```bash
-# add go repository
-zypper ar -f obs://devel:languages:go/ go
-# install latest telegraf
-zypper in telegraf
-```
-
-{{% telegraf/verify %}}
+To use the `zypper` package manager to install the latest stable version of Telegraf, follow these steps:
 
-## Configuration
-
-### Create a configuration file with default input and output plugins
+1.  In your terminal, enter the following command to add the Go repository to the `zypper` configuration:
 
-Every plugin will be in the file, but most will be commented out.
+    ```bash
+    # add go repository
+    zypper ar -f obs://devel:languages:go/ go
+    ```
 
-```
-telegraf config > telegraf.conf
-```
+2.  Enter the following command to install `telegraf`.
 
-### Create a configuration file with specific inputs and outputs
-```
-telegraf --input-filter <pluginname>[:<pluginname>] --output-filter <outputname>[:<outputname>] config > telegraf.conf
-```
+    ```bash
+    # install latest telegraf
+    zypper in telegraf
+    ```
 
-For more advanced configuration details, see the
-[configuration documentation](/telegraf/v1/administration/configuration/).
 {{% /tab-content %}}
 <!---------- BEGIN FreeBSD/PC-BSD ---------->
 {{% tab-content %}}
 Telegraf is part of the FreeBSD package system.
-It can be installed by running:
+
+To use the `pkg` package manager to install the latest stable version of Telegraf, enter the following command:
 
 ```bash
 sudo pkg install telegraf
 ```
 
-The configuration file is located at `/usr/local/etc/telegraf.conf` with examples in `/usr/local/etc/telegraf.conf.sample`.
+The `telegraf` configuration file is installed at `/usr/local/etc/telegraf.conf`.
+Examples are installed at `/usr/local/etc/telegraf.conf.sample`.
 
-{{% telegraf/verify %}}
+{{% /tab-content %}}
+<!---------- BEGIN MACOS ---------->
+{{% tab-content %}}
+Choose from the following options to install Telegraf for macOS:
 
-## Configuration
+- To manually install Telegraf from a file, see the [downloads page](https://portal.influxdata.com/downloads/).
+- [Install using Homebrew](#install-using-homebrew)
 
-### Create a configuration file with default input and output plugins.
+### Install using Homebrew
 
-Every plugin will be in the file, but most will be commented out.
+Users of macOS 10.8 and higher can install Telegraf using the [Homebrew](http://brew.sh/) package manager.
 
-```
-telegraf config > telegraf.conf
-```
+{{% note %}}
```

Comment:
@sanderson Suggestions to say this better?

Reply:
I think this is fine as-is. Looks good.